app/views/editor/ai-chat-message/AIChatMessageCloneModal.js

```
@@ -0,0 +1,157 @@
+      format: 'scenario',
+      links: [{ rel: 'db', href: '/db/ai_scenario/{{$}}', model: 'AIScenario' }],
+    },
+  },
+}
+
+let AIChatMessageCloneView
+
+module.exports = (AIChatMessageCloneView = (function () {
+  AIChatMessageCloneView = class AIChatMessageCloneView extends ModalView {
+    static initClass () {
+      this.prototype.id = 'editor-ai-chat-message-edit-view'
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)